config/main.py

```diff
@@ -85,6 +85,22 @@ def interface_alias_to_name(interface_alias):
     return None
 
 
+def interface_name_is_valid(interface_name):
+    """Check if the interface name is valid
+    """
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    port_dict = config_db.get_table('PORT')
+
+    if interface_name is not None:
+        if not port_dict:
+            click.echo("port_dict is None!")
+            raise click.Abort()
+        for port_name in natsorted(port_dict.keys()):
```

Comment:
Is the natural sort necessary here?

Reply:
Agree. Not necessary. Removed the sorting in other places too.